### DISCUSSION TITLE
Delete an incorrect gps message

### DISCUSSION THREAD

**Discussion by DFOUD** [teslamate-org/teslamate#3567] [Q&A] [answered]
I have a model 3 and after it resets the system the gps seems to go back to USA briefly (I live in China) I tried to remove the id entry represented by states but it doesn't seem to work, I need help thanks!
![QQ截图20231214202721](https://github.com/teslamate-org/teslamate/assets/44726767/21cbff63-b4f8-4e86-8342-5ab323d6d829)
![时间](https://github.com/teslamate-org/teslamate/assets/44726767/a5dc3ad6-9591-4905-889d-563330216626)
![shell](https://github.com/teslamate-org/teslamate/assets/44726767/d81e4dc3-3f50-48a4-880e-724b21cd1991)

**Comment by cwanja** [comment, marked as answer]
You will need to remove the data from the `positions` table to not have the map render the GPS locations.

**Reply by DFOUD** [reply]
I found this link to find the wrong driving record, but the driving page doesn't show the ID I want to delete, can I just delete it?[https://github.com/teslamate-org/teslamate/discussions/2699#discussioncomment-3006475](url)
I'm sure he's right between 4652 and 4655 but the driving page doesn't show 4653 and 4654，
![image](https://github.com/teslamate-org/teslamate/assets/44726767/6624e637-95c5-42c6-b8d6-e4f9625c38c2)
![image](https://github.com/teslamate-org/teslamate/assets/44726767/fa1d5186-88e6-45f5-8ff1-5f2b5a8252bf)

**Reply by DFOUD** [reply]
Yes I managed to find these latitude and longitude messages and delete them!
![image](https://github.com/teslamate-org/teslamate/assets/44726767/1650a2f7-d6d4-4a4c-bf79-d01c5bbb407c)
![image](https://github.com/teslamate-org/teslamate/assets/44726767/1f117bd4-d8c2-45bb-ad8c-d67323985811)
![image](https://github.com/teslamate-org/teslamate/assets/44726767/2f192352-e667-45ef-bdbc-a16c4e731349)

**Comment by DrMichael** [comment]
Can it be closed then?

**Reply by DFOUD** [reply]
Yes I found and deleted the wrong latitude and longitude info.thanks.

**Comment by Etigro** [comment]
Sorry for the newbie question. How did you access the position table ?EGLe 17 déc. 2023 à 11:21, DFOUD ***@***.***> a écrit :﻿
Yes I managed to find these latitude and longitude messages and delete them!
image.png (view on web)
image.png (view on web)
image.png (view on web)

**Reply by cwanja** [reply]
You would need to drop into the database. See the links posted above and here about how to access the database. 

https://docs.teslamate.org/docs/maintenance/manually_fixing_data

**Reply by Etigro** [reply]
Thank you for bearing with me. I am somewhat familiar with the commands in the docs to delete drive and charges. I have done this a few time. What I am not familiar with is how to drop into the database.

**Reply by cwanja** [reply]
Follow [step 1](https://docs.teslamate.org/docs/maintenance/manually_fixing_data/#delete-a-drive-or-charge) under **"Delete a drive or charge"** to drop into the database. Then query the `positions` table to find the erroneous positions. Run a delete command to remove those position IDs.

**Reply by Etigro** [reply]
I always get a fatal error : runtime: out of memory when running step 1

**Reply by cwanja** [reply]
> I always get a fatal error : runtime: out of memory when running step 1

Try and restart the containers and or the host device.

**Reply by Etigro** [reply]
All good, I was able to access the Db and find and delete the bad GPS positions. Thanks a lot.

**Comment by DrMichael** [comment]
Can that be closed?
